mscore/CMakeLists.txt

```diff
@@ -106,16 +106,9 @@ QT5_WRAP_UI (ui_headers
       widgets/font_style_select.ui
       )
 
-if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
-      find_package(Qt5QuickCompiler)
-      qtquick_compiler_add_resources (qrc_files musescore.qrc
-            qml.qrc
-            shortcut.qrc)
-else (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
-      QT5_ADD_RESOURCES (qrc_files musescore.qrc
-            qml.qrc
-            shortcut.qrc)
-endif (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
+QT5_ADD_RESOURCES (qrc_files musescore.qrc
```

Comment:
Why is here `QT5_ADD_RESOURCES` instead of `qtquick_compiler_add_resources`, like in effects/CMakeLists.txt and framework/telemetry/CMakeLists.txt? (I'm just interested, I don't know very much about CMake...)

Reply:
Oops, picked the wrong part it seems